adapta/storage/query_enabled/_qes_delta.py

```diff
@@ -0,0 +1,60 @@
+"""
+ QES implementations for delta-rs.
+"""
+import re
+from dataclasses import dataclass
+from pydoc import locate
+from typing import final, Union, Iterator
+
+import pandas
+from dataclasses_json import DataClassJsonMixin
+
+from adapta.storage.delta_lake import load
+from adapta.storage.models.base import DataPath
+from adapta.storage.models.filter_expression import Expression
+from adapta.storage.query_enabled._models import QueryEnabledStore, CONNECTION_STRING_REGEX
+
+
+@dataclass
+class DeltaCredential(DataClassJsonMixin):
+    """
+    Delta-rs credential helper for QES.
+    """
+
+    auth_client_class: str
+
+    def __post_init__(self):
+        if not self.auth_client_class:
+            raise ValueError("Authentication plugin class name not provided but is required")
+
+
+@dataclass
+class DeltaSettings(DataClassJsonMixin):
+    """
+    Delta QES has no additional settings.
+    """
+
+
+@final
+class DeltaQes(QueryEnabledStore[DeltaCredential, DeltaSettings]):
```

Comment:
I don't like these kind of abbreviations so much. Maybe `DeltaStore` instead? Also `AstraStore` etc.

Reply:
> DeltaQueryEnabledStore